`make html-noplot` instructions outdated

I think the instructions
```
html-noplot:
    $(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
    @echo
    @echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
```
given on https://sphinx-gallery.github.io/stable/configuration.html#without-execution are outdated for the newer versions of sphinx which have a catch-all target.

In my case it resulted in 
```
docs/> make html-noplot
sphinx-build -D plot_gallery=0 -b html  _build/html
usage: sphinx-build [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
sphinx-build: error: the following arguments are required: outputdir, filenames
Makefile:15: recipe for target 'html-noplot' failed
make: *** [html-noplot] Error 2
```

The following would work (an adaption of the catch-all line):
```
html-noplot:
	@$(SPHINXBUILD) -M html "$(SOURCEDIR)" "$(BUILDDIR)" -D plot_gallery=0 $(SPHINXOPTS) $(O)
```
but I don't know if there is a better/preferred way for this.

--- a/doc/configuration.rst
+++ b/doc/configuration.rst
@@ -818,7 +818,7 @@ your ``Makefile`` with:
 .. code-block:: Makefile
 
     html-noplot:
-        $(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+        $(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(SOURCEDIR) $(BUILDDIR)/html
         @echo
         @echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 

